packages/editor/src/components/post-publish-panel/test/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { PostPublishPanel } from '../index';
+
+describe( 'PostPublishPanel', () => {
+	it( 'should render the pre-publish panel if the post is not saving, published or scheduled', () => {
+		const wrapper = shallow(
+			<PostPublishPanel
+				isPublished={ false }
+				isScheduled={ false }
+				isSaving={ false }
+			/>
+		);
+		expect( wrapper ).toMatchSnapshot();
+	} );
+
+	it( 'should render the pre-publish panel is post status is scheduled but date is before now', () => {
```

Comment:
Typo: `s/is post status/if post status/`

Reply:
ae84bd4